### ISSUE TITLE
Data availability

### ISSUE THREAD

**Issue by Peter9192** [seshing/OpenFACADES#6]
Congrats on this nice repo and paper! It's really awesome. 

In the paper you mention 

> Finally, we apply the pipeline at scale by generating labels and captions for half a million buildings in seven cities, laying a foundation for future urban analyses

I'm wondering whether this dataset is also published somewhere. I only found your training data on hugging face.

Secondly, you have a beautiful GIF in your README that shows a 3d building dashboard with metadata popups on hover. I'm wondering whether the software underpinning this dashboard is available somewhere.

Thanks!

**Comment by ycdrn** [comment]
Hi, great work! Congratulations!

I wanted to ask whether you are planning to share the validation dataset mentioned in the paper:

> the building imagery collected from Brussels

It would be very helpful for benchmarking different approaches on the same data.

Thanks a lot!
